modules/nf-core/parabricks/haplotypecaller/main.nf

```diff
@@ -0,0 +1,60 @@
+process PARABRICKS_HAPLOTYPECALLER {
+    tag "$meta.id"
+    label 'process_high'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "Parabricks module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    container "nvcr.io/nvidia/clara/clara-parabricks:4.0.1-1"
```

Comment:
@maxulysse once you are back, I think this container also needs to mirrored to quay.io to handle container registry handeling